### PR TITLE
Add docs for ZNC image

### PR DESCRIPTION
**Pull request by DarthGandalf**
For https://github.com/docker-library/official-images/pull/2617

### CODE REVIEW

**Comment by tianon** [comment]
Looks like this could use some `markdownfmt` to appease Travis -- the easiest way to apply it is via our `markdownfmt.sh` script in the root of this repo, but I'm also happy to apply the reformatting for you, if you'd like. :+1:

File: znc/content.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+```
+
+To run ZNC:
+
+```console
+$ docker run -p 6697:6697 -v znc-cfg:/znc-data znc
+```
+
+The port should match the port you used during `--makeconf`. Note that 6667 is
+often blocked by web browsers, and therefore is not recommended.
```
Comment (tianon) [inline znc/content.md]: Not sure what you mean here by "blocked by web browsers" -- maybe ISPs or firewalls?

File: znc/content.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+If you use any external module, put the .cpp, .py or .pm file to
+`/znc-data/modules` (you may need to create that directory).
+
+IPv6 is currently disabled in this image.
```
Comment (tianon) [inline znc/content.md]: Do you think it'd be worthwhile to expound a little bit about why here, similar to the notes in the `Dockerfile` comments?

File: znc/content.md (hunk 1 above)
Comment (DarthGandalf) [inline znc/content.md]: http://superuser.com/questions/188006/how-to-fix-err-unsafe-port-error-on-chrome-when-browsing-to-unsafe-ports
https://jazzy.id.au/2012/08/23/why_does_chrome_consider_some_ports_unsafe.html

Comment (tianon) [inline znc/content.md]: Oh wow, that's adorable -- I had no idea.  Apologies! :+1:

Comment (DarthGandalf) [inline znc/content.md]: Yeah, we had users complaining all the time that they can't connect, until ZNC itself started warning users during `--makeconf` that 6667 is not the best choice if they want to use a browser.

**Comment by DarthGandalf** [comment]
Done.

**Comment by tianon** [comment]
Tested locally since Travis is having webhook delays due to the GitHub/S3 cascade:
```console
$ curl -fsSL 'https://raw.githubusercontent.com/DarthGandalf/docker-docs/67f571b160c7080f02ec5489a16841dabc37e2f4/znc/content.md' | ./markdownfmt.sh -d
$ curl -fsSL 'https://raw.githubusercontent.com/DarthGandalf/docker-docs/67f571b160c7080f02ec5489a16841dabc37e2f4/znc/license.md' | ./markdownfmt.sh -d
$ curl -fsSL 'https://raw.githubusercontent.com/DarthGandalf/docker-docs/67f571b160c7080f02ec5489a16841dabc37e2f4/znc/variant-slim.md' | ./markdownfmt.sh -d
$ 
```

**Pull request merged**